src/dex/curve-v1/curve-v1.ts

```diff
@@ -681,41 +695,46 @@ export class CurveV1 extends SimpleExchange implements IDex<CurveV1Data> {
     blockNumber: number,
     // list of pool identifiers to use for pricing, if undefined use all pools
     limitPools?: string[],
-    // from: Token,
-    // to: Token,
-    // amounts: bigint[],
-    // side: SwapSide,
-    // routeID: number,
-    // usedPools: { [poolIdentifier: string]: number } | null,
-    // version: string,
-    // isAmountsLinear: boolean,
-    // excludedPools: string[],
-    // // Last two are useful for testing purposes
-    // useCache: boolean,
-    // useEvent: boolean,
+    transferFees: TransferFeeParams = {
+      srcFee: 0,
+      destFee: 0,
+      srcDexFee: 0,
+      destDexFee: 0,
+    },
   ): Promise<ExchangePrices<CurveV1Data> | null> {
     try {
-      if (side === SwapSide.BUY) {
+      const isSell = side === SwapSide.SELL;
+      if (!isSell) {
         return null;
       }
-      const from = this.dexHelper.config.wrapETH(_from);
-      const to = this.dexHelper.config.wrapETH(_to);
+
+      const _isTokenTransferFeeToBeExchanged =
+        isTokenTransferFeeToBeExchanged(transferFees);
```

Comment:
I think we don't care about whether destToken has fees, only srcToken.

Reply:
Yeah, fixed this as well